quinn/src/lib.rs

```diff
@@ -37,6 +37,58 @@
 //! authority. If this is infeasible--for example, if servers are short-lived or not associated
 //! with a domain name--then as with TLS, self-signed certificates can be used to provide
 //! encryption alone.
+//! 
+//! # Feature flags
+//!
+//! ### Crypto features
+//!
+//! These features control which crypto providers quinn will use. Currently rustls is the only one baked into this crate.
+//! See [quinn-boring](https://github.com/quinn-rs/quinn-boring/) for an example of integrating another crypto provider.
+//!
+//! - `rustls`: Enable rustls as the crypto provider for quinn.
+//!
+//! ### Miscellaneous features
+//!
+//! Miscellaneous optional features.
```

Comment:
Nit: this line doesn't seem to provide much information.

Reply:
Resolved by [e6d87f7](https://github.com/quinn-rs/quinn/pull/2426/commits/e6d87f7310b15527fd6afc78d7288f2b4f19b750)